lib/features/impact_groups/repo/impact_groups_repository.dart

```diff
@@ -69,35 +63,17 @@ class ImpactGroupsRepositoryImpl with ImpactGroupsRepository {
   List<ImpactGroup>? _impactGroups;
 
   Future<void> _init() async {
-    _givtRepository.onGivtsChanged().listen(
-          (_) => _fetchImpactGroups(),
-        );
     _createFamilyGoalRepository.onFamilyGoalCreated().listen(
           (_) => _fetchImpactGroups(),
         );
     _parentalApprovalRepository.onParentalApprovalChanged().listen(
           (_) => _fetchImpactGroups(),
         );
-    _authRepository.hasSessionStream().listen(
```

Comment:
Im struggling to think how to see the difference because we dont really display group related things anymore. But in code I think yes when switching between US accounts the groups dont get cleared

Reply:
We also use it for the parent invite. So a "convoluted" scenario would be:
- Create an account, then logout with that account
- Open an existing account, invite the newly created account
- Logout and then login with the newly created account, if the parent invite doesn't appear we got a bug